From 47f6f53cd39997f8e7fea5235b27a30fbdc9dd4d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20K=2E=20H=C3=BCttel?= <dilfridge@gentoo.org>
Date: Sat, 23 Dec 2023 22:56:11 +0100
Subject: [PATCH 9/9] Gentoo: Disable valgrind-based tests, too unreliable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas K. Hüttel <dilfridge@gentoo.org>
---
 elf/Makefile | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/elf/Makefile b/elf/Makefile
index ba7b71035c..4337603c6b 100644
--- a/elf/Makefile
+++ b/elf/Makefile
@@ -480,7 +480,6 @@ tests += \
   unload6 \
   unload7 \
   unload8 \
-  valgrind-test \
   # tests
 tests-cxx = \
   tst-dlopen-nodelete-reloc \
@@ -611,9 +610,6 @@ $(objpfx)tst-relro-libc.out: tst-relro-symbols.py $(..)/scripts/glibcelf.py \
 	    --required=__io_vtables \
 	  > $@ 2>&1; $(evaluate-test)
 
-ifeq ($(run-built-tests),yes)
-tests-special += $(objpfx)tst-valgrind-smoke.out
-endif
 $(objpfx)tst-valgrind-smoke.out: tst-valgrind-smoke.sh $(objpfx)ld.so $(objpfx)valgrind-test
 	$(SHELL) $< $(objpfx)ld.so  $(rtlddir)/$(rtld-installed-name) '$(test-wrapper-env)' \
 		'$(run-program-env)' '$(rpath-link)' $(objpfx)valgrind-test > $@; $(evaluate-test)
-- 
2.41.0

